main/lsp/LSPIndexer.cc

```diff
@@ -299,14 +299,12 @@ LSPFileUpdates LSPIndexer::commitEdit(SorbetWorkspaceEditParams &edit) {
         ENFORCE(runningSlowPath.epoch <= pendingTypecheckUpdates.epoch);
         ENFORCE(runningSlowPath.epoch > (pendingTypecheckUpdates.epoch - pendingTypecheckUpdates.editCount));
 
-        auto merged = update.copy();
-        merged.mergeOlder(pendingTypecheckUpdates);
-        merged.canTakeFastPath = canTakeFastPath(merged, true);
-        // Cancel if old + new takes fast path, or if the new update will take the slow path anyway.
-        if ((merged.canTakeFastPath || !update.canTakeFastPath) &&
-            initialGS->epochManager->tryCancelSlowPath(merged.epoch)) {
+        // Cancel if the new update will take the slow path anyway.
```

Comment:
(Not on this line, but this is the closest place that github will allow a comment)

The comment up on line 294 refers to making a copy (which no longer happens down here)...perhaps reword?